extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaFocusWidget.js

```diff
@@ -33,7 +33,8 @@ ve.ui.WikiaFocusWidget = function VeUiWikiaFocusWidget( surface ) {
 	this.$body = this.$( this.getElementDocument() ).find( 'body:first' );
 	this.$window = this.$( this.getElementWindow() );
 	this.$surface = surface.$element;
-	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( h1 )' );
+	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( .header-container  )' ).add( '.header-tally' );
```

Comment:
I don't see a benefit of it, coupling would still remain - as ve would have to know of internals of header (to know which class where to add). Using ve-activated class, as now, at least makes it easy to notice in header code that it styles slightly different for VE.
Ultimately I think it should be handled in a way more manageable way without creating such a strong coupling but I don't have time to handle it now.